datastore/query_test.go

```diff
@@ -500,6 +500,77 @@ var (
 	}
 )
 
+func TestPropertyFilterToProto(t *testing.T) {
+	testCases := []struct {
```

Comment:
suggestion: It looks like there's only one valid test case that you're testing (all other testCases focus on error states). I might add a couple more test cases. For example, it looks like you allow developers to pass in field names that include whitespaces--you should probably add a testCase for that.

Reply:
Done.